GVFS/GVFS.Common/Git/GitProcess.cs

```diff
@@ -714,6 +714,24 @@ public bool HasErrors
             {
                 get { return this.ReturnCode != SuccessCode; }
             }
+
+            public bool HasNonWarningErrors
+            {
+                get
+                {
+                    if (this.HasErrors && this.Errors.Any())
+                    {
+                        return !this.Errors
+                            .Trim()
+                            .Split('\n')
+                            .All(line => line.TrimStart().StartsWith("warning:"));

```

Comment:
Does the `StartsWith` need to be case insensitive?

Reply:
Yeah, good point.